core/domain/user_services_test.py

```diff
@@ -1318,6 +1603,242 @@ def test_record_user_started_state_translation_tutorial(self):
         self.assertTrue(
             user_settings.last_started_state_translation_tutorial is not None)
 
+    def test_get_human_readable_user_ids(self):
+        auth_ids = ['user1', 'user2', 'user3']
+        user_emails = [
+            '1user@example.com',
+            '2user@example.com',
+            '3user@example.com']
+        user_ids = []
+
+        for i, auth_id in enumerate(auth_ids):
+            user_ids.append(user_services.create_new_user(
+                auth_id, user_emails[i]).user_id)
+
+        user_services.set_username(user_ids[0], 'username1')
+        user_services.mark_user_for_deletion(user_ids[1])
+
+        user_settings3 = user_services.get_user_settings(user_ids[2])
```

Comment:
I like that you're testing 3 different cases here. But I think you could perhaps use better variable names and perhaps usernames/emails that convey the semantics, e.g. regular_user, user_being_deleted, etc.

Reply:
Same thing with the variable name, I think the comment isn't actually addressed in the code (user_settings3 is vague and should ideally be more semantic). You might want to check the code as well.

Reply:
Done